great-idea/css/index.css

```diff
@@ -63,4 +63,188 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* start here */
+
+
+/* Header Box */
+
+body .container {
+	margin: 10px 40px;
+}
+
+body .header-box {
+	display:flex;
+	flex-flow: row nowrap;
+	justify-content: space-evenly;
+	align-items: center;
+	margin-bottom: 30px;
+}
+
+.header-box nav a {
+	color: gray;
+	text-decoration: none;
+	margin: 0 20px;
+}
+
+.header-box nav {
+	font-size: 25px;
+	justify-content: space-evenly;	
+}
+
+/* CTA box */
+
+body .cta {
+	display: flex;
+	flex-flow: row wrap;
+	justify-content: space-evenly;
+	align-content: center;
+	margin-bottom: 60px;
+
+}
+
+body .cta-text {
+	font-size: 75px;
+	text-align: center;
+	padding-top: 50px;
+}
+
+body .cta-text button {
+	background: white;
+	vertical-align: top;
+	margin-top: 6px;
+	padding: 8px 44px;
+	font-size: 17px;
+	border: 1px solid black;
+	cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+	background: black;
+	color: white;
+  }
+  
+  body .main-content .top-content {
+	  display: flex;
+	  flex-flow: row nowrap;
+	  justify-content: center;
+	  align-content: center;
+  }
+
+  body .main-content .text-content {
+	  padding: 40px 0;
+	  
+  }
+
+  body .main-content img {
+	  width: 100%;
+  }
+
+  body .main-content .bottom-content {
+	display: flex;
+	flex-flow: row nowrap;
+	justify-content: center;
+	align-content: center;
+}
+
+body .main-content {
+	border-top:1px solid black;
+	border-bottom: 1px solid black;
+	padding-bottom: 20px;
+}
+
+/* contact */
+
+body .contact {
+	display: flex;
+	flex-direction:column;
+	justify-content: space-around;
+	border: 1px solidblack;
+	margin-top: 40px;
+	margin-bottom: 30px;
+}
+
+.contact h4, p {
+	height: 45px;
+}
+
+/* footer */
+
+body footer {
+	height: 50px;
+	margin-top: 40px;
+	text-align: center;
+	margin-bottom: 50px;
+}
+
+
+/* New CSS from Services Page */
+
+body .services-header-img {
+	width:100%;
+}
+
+body .services {
+	border-bottom: 1px solid black;
+	padding: 30px 0;
+}
+
+body .services h2 {
```

Comment:
I would make this either an H1 or increase the font-size. You'll want this bigger than your nav links up top